src/asynk/async_queue.rs

```diff
@@ -608,7 +727,9 @@ mod async_queue_tests {
         assert_eq!(Some(1), number);
         assert_eq!(Some("AsyncTask"), type_task);
 
-        let task = insert_task(&mut transaction, &AsyncTask { number: 2 }).await;
+        let task = AsyncTask { number: 2 };
+        let metadata = serde_json::to_value(&task as &dyn AsyncRunnable).unwrap();
```

Comment:
you're using the same steps to insert tasks in tests, maybe it's worth returning the `insert_task` helper?

Reply:
Done in [ca8c2ad](https://github.com/ayrat555/fang/pull/46/commits/ca8c2ad4144436f3f28e5600710d30bf392f8867)